crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -41,25 +41,42 @@ impl WebpackLoadersOptions {
 #[turbo_tasks::value(shared)]
 #[derive(Default, Clone)]
 pub struct ModuleOptionsContext {
+    #[serde(default)]
```

Comment:
Are the `serde(default)`s necessary if the struct implements `Default` itself?

Reply:
Yes, need it